polysemy-zoo-0.1.2.0 test suite build failure

As seen on the Stackage build server:

```
[3 of 6] Compiling MTLSpec          ( test/MTLSpec.hs, dist/build/polysemy-zoo-test/polysemy-zoo-test-tmp/MTLSpec.o )

test/MTLSpec.hs:74:11: error:
    • Ambiguous use of effect 'Writer'
      Possible fix:
        add (Member (Writer [a0]) '[Writer [Int]]) to the context of
          the type signature
      If you already have the constraint you want, instead
        add a type application to specify
          '[a0]' directly, or activate polysemy-plugin which
            can usually infer the type correctly.
    • In a stmt of a 'do' block: tell [1]
      In the second argument of ‘($)’, namely
        ‘do tell [1]
            absorbWriter $ replicateTell 2 5
            censor init $ absorbWriter $ replicateTell 2 5’
      In a stmt of a 'do' block:
        flip shouldBe 16 . sum . fst . run . runWriter
          $ do tell [1]
               absorbWriter $ replicateTell 2 5
               censor init $ absorbWriter $ replicateTell 2 5
   |
74 |           tell [1]
   |           ^^^^^^^^

test/MTLSpec.hs:76:11: error:
    • Ambiguous use of effect 'Writer'
      Possible fix:
        add (Member (Writer [a1]) '[Writer [Int]]) to the context of
          the type signature
      If you already have the constraint you want, instead
        add a type application to specify
          '[a1]' directly, or activate polysemy-plugin which
            can usually infer the type correctly.
    • In the expression: censor init
      In a stmt of a 'do' block:
        censor init $ absorbWriter $ replicateTell 2 5
      In the second argument of ‘($)’, namely
        ‘do tell [1]
            absorbWriter $ replicateTell 2 5
            censor init $ absorbWriter $ replicateTell 2 5’
   |
76 |           censor init $ absorbWriter $ replicateTell 2 5
   |           ^^^^^^^^^^^
```

I was able to reproduce this locally like so:

```bash
stack unpack polysemy-zoo-0.1.2.0 && cd polysemy-zoo-0.1.2.0
echo 'resolver: nightly-2019-06-08' > stack.yaml
stack build --test --bench --no-run-benchmarks --fast
```

--- a/stack.yaml
+++ b/stack.yaml
@@ -4,8 +4,8 @@ packages:
 - .
 
 extra-deps:
-- polysemy-0.3.0.0
-- polysemy-plugin-0.2.0.0
+- polysemy-0.3.0.1
+- polysemy-plugin-0.2.0.2
 - th-abstraction-0.3.1.0
 - bifunctors-5.5.4
 

--- a/test/MTLSpec.hs
+++ b/test/MTLSpec.hs
@@ -1,6 +1,7 @@
 {-# LANGUAGE OverloadedStrings   #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeApplications    #-}
+
 module MTLSpec where
 
 import           Polysemy
@@ -70,7 +71,7 @@ spec = describe "MTL" $ do
       ++ " In this case, 16, the sum of \"init [1,5,5,5,5]\""
       )
     $ do
-        flip shouldBe 16 . sum . fst . run . runWriter $ do
+        flip shouldBe 16 . sum @[] . fst . run . runWriter $ do
           tell [1]
           absorbWriter $ replicateTell 2 5
           censor init $ absorbWriter $ replicateTell 2 5

